pandas/tests/frame/test_alter_axes.py

```diff
@@ -641,6 +641,36 @@ def test_reset_index(self):
         xp = xp.set_index(['B'], append=True)
         assert_frame_equal(rs, xp, check_names=False)
 
+    def test_reset_index_level(self):
+        df = pd.DataFrame([[1, 2, 3, 4], [5, 6, 7, 8]],
+                          columns=['A', 'B', 'C', 'D'])
+
+        # With MultiIndex
+        for levels in ['A', 'B', 'E'], [0, 1, 2]:
```

Comment:
Can you remove the 'E' and 2 from the lists? I founded it a bit confusing as you don't use it